frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -1,13 +1,19 @@
+/* eslint-disable */
+// TODO work around eslint error WithTranslation not found in 'react-i18next'
```

Comment:
Is this TODO still needed?

Reply:
Updated it 
`Work around eslint error WithTranslation not found in 'react-i18next'`